VATSpy.dat

```diff
@@ -18222,17 +18222,25 @@ LGMD-W|Makedonia ACC (West) - Athinai|LGMD_W|LGMD-W
 LGMD-LMO|Makedonia ACC (Limnos) - Athinai|LGMD_LMO|LGMD-LMO
 LGMD-KVL|Makedonia ACC (Kavala) - Athinai|LGMD_KVL|LGMD-KVL
 LHCC|Budapest||LHCC
-LIBB|Brindisi ACC - Brindisi||LIBB
-LIMM-ES|Milano ACC ES|LIMM_ES|LIMM-ES
-LIMM-WN|Milano ACC WN|LIMM_WN|LIMM-WN
-LIMM-WS|Milano ACC WS|LIMM_WS|LIMM-WS
-LIMM|Milano ACC EN|LIMM_EN|LIMM
-LIPP-SD|Padova ACC SD|LIPP_SD|LIPP-SD
-LIPP-CE|Padova ACC CE|LIPP_CE|LIPP-CE
-LIPP|Padova ACC NE|LIPP_NE|LIPP
-LIRR-NC|Roma ACC NC - Roma|LIRR_NC|LIRR-NC
-LIRR-S|Roma ACC S - Roma|LIRR_S|LIRR-S
-LIRR|Roma ACC N - Roma|LIRR_N|LIRR
+LIBB|Brindisi ACC||LIBB
+LIMM|Milano ACC - EN|LIMM_EN|LIMM
+LIMM-ES|Milano ACC - ES|LIMM_ES|LIMM-ES
+LIMM-WN|Milano ACC - WN|LIMM_WN|LIMM-WN
+LIMM-WS|Milano ACC - WS|LIMM_WS|LIMM-WS
+LIPP|Padova ACC - NE|LIPP_NE|LIPP
+LIPP-SD|Padova ACC - SD|LIPP_SD|LIPP-SD
+LIPP-CE|Padova ACC - CE|LIPP_CE|LIPP-CE
+LIRR|Roma ACC - NE - Roma|LIRR_NE1|LIRR

```

Comment:
LIRR_NE1 entry not required here - any logon not defined below will automatically revert to the whole LIRR FIR